modules/microshift-greenboot-included-health-checks.adoc

```diff
@@ -0,0 +1,21 @@
+//Module included in the following assemblies:
+//
+//* microshift_running-applications/microshift-greenboot-workload-scripts.adoc
+
+:_content-type: CONCEPT
+[id="microshift-greenboot-included-health-checks_{context}"]
+= Included greenboot health checks
+
+Health check scripts are available in `/usr/lib/greenboot/check`, a read-only directory in RPM-OSTree systems. The following health checks are included with `greenboot-default-health-checks`.
```

Comment:
```suggestion
Health check scripts are available in `/usr/lib/greenboot/check`, a read-only directory in RPM-OSTree systems. The following health checks are included with `greenboot-default-health-checks` framework.
```

Reply:
I'm not sure if that is correct, but I think it needs a noun to help identify the object reference.